Pipfile

```diff
@@ -0,0 +1,74 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
They should be all be put in `dev-packages` this library has no direct dependencies.
I would only keep the main depencencies and let pip handle the dependencies of dependencise, so you should only have the followings:

- pylint
- isort
- yapft
- coverage
- django
- flask
- nose2
- packaging
- requests
- twine

I might have forgotten some, needs to be tested. I would also only fix the minor version so using `~=` instead of `==` as best practice.